src/parser/BinaryTokenizer.cc

```diff
@@ -164,6 +174,35 @@ Parser::BinaryTokenizer::area(uint64_t size, const char *description)
     return result;
 }
 
+Ip::Address
+Parser::BinaryTokenizer::inV4(const char *description)
+{
+    struct in_addr addr;
+    const auto size = sizeof(addr);
+    want(size, description);
+    const auto rawData = data_.substr(parsed_, size);
+    memcpy(&addr, rawData.rawContent(), size);
```

Comment:
If possible, avoid creating `rawData` to minimize SBuf noise:
```suggestion
    memcpy(&addr, data_.rawContent() + parsed_, size);
```